python/hyperspace/tests/test_indexmanagement.py

```diff
@@ -89,6 +95,7 @@ def test_index_metadata(self):
         df = self.hyperspace.index('idx1')
         df.show()
 
+
```

Comment:
nit: --

Reply:
according to PEP8, (common guildline for python formatting)

https://www.python.org/dev/peps/pep-0008/#blank-lines  says that


> Surround top-level function and class definitions with two blank lines. 

And this is default behavior of python formatters such as black formatter.